nav2_bringup/bringup/worlds/waffle.model

```diff
@@ -137,6 +137,7 @@
             </linear_acceleration>
           </imu>
           <plugin name="turtlebot3_imu" filename="libgazebo_ros_imu_sensor.so">
+            <initial_orientation_as_reference>false</initial_orientation_as_reference>
```

Comment:
This change is unrelated to the ticket, please revert

Reply:
I don't know how this line made its way into this. I certainly didn't add this intentionally. 

I remove it. This should now be ready for merge.